### PR TITLE
Sort members of `VotingList` alphabetically

### PR DESCRIPTION
**Pull request by linusha**
This should fix the issue everywhere where we use `->members` of a `VotingList`. Fixes e.g., #751.

The testing of this is kind of meh.
We could either make a real VotingListController Test which would make sense imo, or beef the other two or three tests where this is used up as well. Especially the votinglist/show test. However, to do this well I would have needed to create `expectSelectorTextInOrder`.

### CODE REVIEW

**Comment by tillprochaska** [comment]
Maybe it would make sense to use a [global scope](https://laravel.com/docs/9.x/eloquent#global-scopes) to ensure that members are always ordered alphabetically (if not explicitly ordered otherwise)?

**Comment by linusha** [comment]
> Maybe it would make sense to use a [global scope](https://laravel.com/docs/9.x/eloquent#global-scopes) to ensure that members are always ordered alphabetically (if not explicitly ordered otherwise)?

Nice, I forgot about those :slightly_smiling_face:

File: app/app/Http/Controllers/VotingListsController.php [hunk 1]
```diff
@@ -155,7 +155,7 @@ public function json(VotingList $votingList)
         return response()->json($vote);
     }
 
-    private function members(VotingList $votingList)
+private function members(VotingList $votingList)
```
Comment (tillprochaska) [inline app/app/Http/Controllers/VotingListsController.php:158]: Unintended change?

**Review by tillprochaska — APPROVED** [review]
Nice 👍 There’s a weird formatting change in one file, I guess that’s unintended, apart from that feel free to merge :)

**Pull request closed without merging**